### PR TITLE
Issue/144 convert file tests

### PR DESCRIPTION
**Pull request by sethbattin**
I saw issue #144 fly by my inbox and decided to try to do it.  Let me know if I've missed anything glaring; I've never looked closely at the library before.

Note that my last test method, `testFileCreateRandomKey`, is failing.  I strongly suspect that the method being tested (`File::createNewRandomKey`) is a relic that should be removed.  It has a name that is identical to other code, but it doesn't function the same way.  If I'm right, I could add another commit to remove those parts.

### CODE REVIEW

**Comment by defuse** [comment]
This is awesome. They're even catching new bugs! I'll review this right away :D

File: test/unit/FileTest.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    {
+        $data = [];
+        
+        $data['wat-giagantic-duck'] = ['wat-gigantic-duck.jpg'];
+        $data['large'] = ['large.jpg'];
+        
+        if (file_exists(__DIR__ . '/File/In_the_Conservatory.jpg')){
+            // see File/get_large.sh
+            $data['extra-large'] = ['In_the_Conservatory.jpg'];
+        }
```
Comment (defuse) [inline test/unit/FileTest.php]: I wonder why that file isn't checked in. I'm not even sure where to get it.

Comment (paragonie-scott) [inline test/unit/FileTest.php]: get_large.sh

Comment (defuse) [inline test/unit/FileTest.php]: Oh I'm an idiot... there's a comment right there telling me to look in get_large.sh.

**Comment by defuse** [comment]
Looks good!

File: test/unit/FileTest.php (hunk 1 above)
Comment (paragonie-scott) [inline test/unit/FileTest.php]: It's too large for github. The file tests were originally intended for
benchmarking memory usage BTW. I welcome making these unit tests :)

**Pull request merged**